### PR TITLE
feat(component-testing): New layout for runner

### PR DESCRIPTION
**Pull request by dmtrKovalenko**
This PR changes the layout of the component testing mode, so it unlocks the #8671 

New layout according to [figma mockup](https://www.figma.com/file/JTTy3BX2TjBjCAsJOc0vNF/TR---Unified?node-id=467%3A0):
image

The upcoming PR will include:
- [ ] Updated command log header that removes "back to test" button and collapsing

### CODE REVIEW

File: packages/runner-ct/src/app/BottomPane.jsx [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import * as React from 'react'
+
```
Comment (lmiller1990) [inline packages/runner-ct/src/app/BottomPane.jsx:2]: Can we start to use `tsx` everywhere, at least for all new components? no reason not too

File: packages/runner-ct/src/app/app.jsx [hunk 2]
```diff
@@ -1,169 +1,61 @@
-import cs from 'classnames'
-import { observable } from 'mobx'
 import { observer } from 'mobx-react'
 import PropTypes from 'prop-types'
-import React, { Component } from 'react'
-import { findDOMNode } from 'react-dom'
+import React from 'react'
```
Comment (lmiller1990) [inline packages/runner-ct/src/app/app.jsx:3]: would it be too time consuming to make this into a `tsx` file too?  if not, I think we should go for it.

I think the time investment now to TS all the things will be worth it.

File: packages/runner-ct/src/app/app.jsx [hunk 3]
```diff
@@ -1,169 +1,61 @@
...
 import { Reporter } from '@packages/reporter'
-import { $ } from '@packages/driver'
 
 import errorMessages from '../errors/error-messages'
 import util from '../lib/util'
 import State from '../lib/state'
 
 import SpecsList from '../specs/specs-list'
+import SplitPane from 'react-split-pane'
```
Comment (lmiller1990) [inline packages/runner-ct/src/app/app.jsx:11]: not sure if we have a standard around imports but seems the standard in this file is `node_modules` at the top. but this is not a blocker, just something that I noticed.

**Review by lmiller1990 — COMMENTED** [review]
Left some comments, no real blockers, just some questions/recommendations! looks great

File: packages/runner-ct/src/app/app.jsx (hunk 2 above)
Comment (dmtrKovalenko) [reply packages/runner-ct/src/app/app.jsx:3]: I will be happy to do this in separate PR

File: packages/runner-ct/src/app/BottomPane.jsx (hunk 1 above)
Comment (dmtrKovalenko) [reply packages/runner-ct/src/app/BottomPane.jsx:2]: It's not possible with runner-ct without configuration. Will be happy to migrate to PR in separate PR

**Review by lmiller1990 — APPROVED** [review]
cool, we will TS all the things in another PR

**Review by JessicaSachs — CHANGES_REQUESTED** [review]
Regression in the command log CSS. Can't scroll down if we overflow-y.

Test Case:
1. `yarn cy:open` in Vue package
2. Run advanced/mocking-axios/3-Users.spec.js
3. You can't scroll down -- the stack trace is covered, etc

See pic
Screen Shot 2020-12-16 at 12 48 33 PM

**Comment by dmtrKovalenko** [comment]
Sorry, backported fix of overflow from the next branch

**Comment by lmiller1990** [comment]
LGTM except Percy failing, not sure if that's a blocker, prob not.

**Pull request merged**